topomodelx/nn/cell/can.py

```diff
@@ -129,8 +127,8 @@ def forward(
 
         Returns
         -------
-        torch.Tensor
-            Output prediction for the cell complex.
+        x_1: torch.Tensor, shape = (num_pooled_edges, heads * out_channels)
```

Comment:
Singular return values do not have a name.

Reply:
Done.